edweiss-app/package.json

```diff
@@ -61,6 +61,7 @@
     "expo-system-ui": "~3.0.7",
     "expo-web-browser": "~13.0.3",
     "firebase": "^10.14.1",
+    "firebase-admin": "^13.0.0",
```

Comment:
Uh oh this is not okay, why is the server/admin package added as a dependency inside the client?

Reply:
I don't know It has been added automatically I guess. I will fix it now thank you